.github/workflows/actions.yaml

```diff
@@ -93,38 +93,44 @@ jobs:
         run:  go mod download
       - name: Build karavi-authorization Docker Images
         run: make docker  
+      - name: Setting version variables for job
```

Comment:
Can the name be updated that will add clarity to what this action is doing? For example, something like "Get docker image tags for image scans"

Reply:
I've updated with your suggestion. I agree that sounds better